src/stm32f411/pin.cpp

```diff
@@ -17,10 +17,14 @@
 #include <libhal-arm-mcu/stm32f411/pin.hpp>
 
 #include <libhal-util/bit.hpp>
+#include <libhal-util/enum.hpp>
 #include <libhal/units.hpp>
 
 #include "gpio_reg.hpp"
+#include "libhal-arm-mcu/stm32f411/constants.hpp"
+#include "lpc40/pin_reg.hpp"
 #include "power.hpp"
+#include "stm32f411/rcc_reg.hpp"
```

Comment:
```suggestion
#include "rcc_reg.hpp"
```

Aren't these in the same dir?